modules/node_modules/@ncigdc/components/ProjectVisualizations.js

```diff
@@ -49,24 +48,25 @@ const enhance = compose(
   withState('selectedFrequentMutationsSurvivalData', 'setSelectedFrequentMutationsSurvivalData', {}),
   withState('state', 'setState', initialState),
   withState('numCasesAggByProject', 'setNumCasesAggByProject', undefined),
-  withPropsOnChange(['projectId', 'numCasesAggByProject', 'survivalPlotSampleSize'], async ({ projectId, setState, numCasesAggByProject, survivalPlotSampleSize }) => {
-    if (!numCasesAggByProject) {
-      return;
+  withPropsOnChange(
+    ['projectId', 'caseCount', 'survivalPlotSampleSize'],
+    async ({ projectId, setState, caseCount, survivalPlotSampleSize }) => {
+      if (!projectId) return;
+
+      const defaultSurvivalData = await getDefaultCurve({
+        totalCases: caseCount,
+        currentFilters: { op: '=', content: { field: 'cases.project.project_id', value: projectId } },
```

Comment:
should this also take in the clinical filters like https://github.com/NCI-GDC/portal-ui/blob/relay/modules/node_modules/%40ncigdc/routes/ProjectRoute.js#L29 cuz the ticket says this 
> I believe the Survival plot title should show 974 cases instead of 986 = number of cases for which we have clinical data needed for the analysis (as opposed to number of donors with ssm data)